Makefile.in

```diff
@@ -103,7 +103,7 @@ CONFIG_CLEAN_VPATH_FILES =
 am__installdirs = "$(DESTDIR)$(bindir)"
 PROGRAMS = $(bin_PROGRAMS)
 am_things_n_stuff_OBJECTS = main.$(OBJEXT) things.$(OBJEXT) \
-	stuff.$(OBJEXT)
+	stuff.$(OBJEXT) complex_types.$(OBJEXT)
```

Comment:
I believe Makefile.in is also a generated file. Remove from the repo?

Reply:
removed